packages/verifiable-credentials/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@crossmint/verify-credentials",
     "private": true,
-    "version": "1.1.1",
+    "version": "1.2.0-alpha.1",
```

Comment:
u dont need to edit this manually, just follow https://github.com/Crossmint/crossmint-client-sdk/blob/main/README.md